include/mercury/kes_dashboard_live_tiles_cli/OnlineLoader.h

```diff
@@ -8,32 +8,49 @@
  */
 
 
-#ifndef __KES_DASHBOARD_LIVE_TILES_CLI_ONLINE_LOADER_H__
-#define __KES_DASHBOARD_LIVE_TILES_CLI_ONLINE_LOADER_H__
+#ifndef INCLUDE_MERCURY_KES_DASHBOARD_LIVE_TILES_CLI_ONLINELOADER_H_
+#define INCLUDE_MERCURY_KES_DASHBOARD_LIVE_TILES_CLI_ONLINELOADER_H_
 
 
 #include <list>
+#include <memory>
 #include <string>
 
 #include "mercury/_http/http_client.h"
+#include "mercury/kes_dashboard_live_tiles_cli/ITile.h"
 #include "mercury/kes_dashboard_live_tiles_cli/ITileLoader.h"
 
+using std::list;
+using std::shared_ptr;
+using std::string;
+
+
+#define ONE_HOUR 1000 * 60 * 60  // in milliseconds
```

Comment:
Better to use `const int` or `constexpr int`, etc in this scenario as you get type checking when you use it.